src/EnergyPlus/Plant/Component.cc

```diff
@@ -129,5 +129,13 @@ namespace DataPlant {
         this->compPtr->simulate(state, this->location, FirstHVACIteration, this->MyLoad, this->ON);
     }
 
+    void CompData::oneTimeInit(EnergyPlusData &state) const
+    {
+        if (this->compPtr->oneTimeInitFlag) {
+            this->compPtr->oneTimeInit_new(state);
```

Comment:
Had to create a new oneTimeInit_new function call because if I called the original oneTimeInit here, it would create math diffs.